ros2_ws/src/rcdt_utilities/nodes/transform_pose_node.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Alliander N. V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import rclpy
+from copy import copy
+from rclpy import time
+from rclpy import logging
+from rclpy.node import Node
+from tf2_ros import TransformException
+from tf2_ros.buffer import Buffer
+from tf2_ros.transform_listener import TransformListener
+from tf2_geometry_msgs import do_transform_pose_stamped
+from geometry_msgs.msg import PoseStamped, Transform, TransformStamped
+from rcdt_utilities_msgs.srv import TransformPose
+
+ros_logger = logging.get_logger(__name__)
+
+
+class TransformPoseNode(Node):
```

Comment:
Deze service opsplitsen naar een transform en een transpose (of iets in die richting)